src/main/java/com/drajer/bsa/model/HealthcareSetting.java

```diff
@@ -49,7 +49,7 @@
   /**
    * The attribute represents the client secret that is to be used for SMART on FHIR Authorization.
    */
-  @Column(name = "clientSecret", nullable = true, columnDefinition = "TEXT")
+  @Column(name = "clientSecret", columnDefinition = "TEXT")
```

Comment:
What happened to all the `nullable = true`?

Reply:
nullable is true by default